packages/components/data-table/src/data-table.styles.tsx

```diff
@@ -55,7 +55,7 @@ const TableContainer = styled.div<TTableContainer>`
 
   ${(props) =>
     // this is needed in order to have a sticky header
-    props.maxHeight ? `overflow-x: auto;` : ''}
+    props.maxHeight ? `overflow-x: auto;` : 'overflow-x: scroll;'}
```

Comment:
```suggestion
overflow-x: auto;
```

In theory, `auto` is solving the same problem as `scroll` does, but in a more elegant way. Introducing scrollbars only when needed instead of having them visible at all times. This should be sufficient, if it is a scrolling-issue.

Reply:
Oh yes, I realised and moved it back to wip. Thank you 🙏🏾